src/main/java/com/crowd/service/ProjectService.java

```diff
@@ -97,11 +84,36 @@ public int createProject(UserBean user, String projectName, double needAmount, S
         project.setNeedAmount(needAmount);
         project.setAboutProject(aboutProject);
         project.setUser(userDAO.findByEmail(user.getEmail()));
-        project.setStatus("Actual");
+        project.setStatus(Status.NOT_STARTED);
         project.setDate(new Timestamp(System.currentTimeMillis()));
         project.setUrl(url);
         projectDAO.insert(project);
 
         return project.getId();
     }
+
+
+    public void checkProjectStatus(Project project){
+        switch (project.getStatus()) {
+            case NOT_STARTED:  {
+                if (project.getDonate_amount()>=project.getNeedAmount()){
+                    project.setStatus(Status.FUNDED);
+                    update(project);
+                }else if (project.getDonate_amount()>0){
+                    project.setStatus(Status.IN_PROGRESS);
+                    update(project);
```

Comment:
Refactor this code in order to remove duplication - you may extract method that will return new status for the project.